tests/test_apigw.py

```diff
@@ -604,6 +604,27 @@ def test_set_wafv2_active_response_tag_resource(self):
         resources = policy.push(event_data("event-cloud-trail-tag-rest-stage.json"))
         self.assertEqual(len(resources), 1)
 
+    def test_config_id_stagearn_response(self):
+        factory = self.replay_flight_data("test_rest_stage_config_id")
+        p = self.load_policy(
+            {
+                "name": "config_id-apigw-rest-stage-check",
+                "resource": "rest-stage",
+                "mode": {
+                    "type": "config-poll-rule",
+                    "role": "arn:aws:iam::{account_id}:role/MyRole",
+                    "schedule": "TwentyFour_Hours",
+                    "ignore-support-check": True
+                }
+            },
+            session_factory=factory,
+            config={'region': 'us-west-2'},
+            validate=False
+        )
+        event = event_data('poll-evaluation.json', 'config')
+        results = p.push(event, None)
+        self.assertEqual(len(results), 1)
```

Comment:
@kk1532 i don't think this actually tests the new logic that we've adding. This assert only checks that the number of matched resources is what we expect. It doesn't check that we're using the correct value for resource id when sending evaluations to Config. I actually don't know how we can do that. However, maybe we can at least check that the 'stageArn' attribute is set correctly from the augment method.

Reply:
@darrendao I have added new test code for checking `stageArn` addition. Can you please review it?

Reply:
LGTM